### PR TITLE
Kata 7, "Collatz Conjecture Length".

### PR DESCRIPTION
**Pull request by udodikdmitro**

### CODE REVIEW

File: test/katatraining/collatzcojnjecturelength/CollatzTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public void testSomething() {
+        assertEquals("Test failed for: " + 100L, 26L, Collatz.conjecture(100L));
+        assertEquals("Test failed for: " + 10L, 7L, Collatz.conjecture(10L));
+        assertEquals("Test failed for: " + 500L, 111L, Collatz.conjecture(500L));
+        assertEquals("Test failed for: " + 1000000000L, 101L, Collatz.conjecture(1000000000L));
+        assertEquals("Test failed for: " + 1000000000000000L, 276L, Collatz.conjecture(1000000000000000L));
+    }
+
+    private void assertEquals(String s, long l, long conjecture) {
+    }
```
Comment (ezseeton) [inline test/katatraining/collatzcojnjecturelength/CollatzTest.java]: What is it? This method is empty and was created only to compilation will be passed?

This method, assertEquals should be part of org.junit import. Please don't create something just to hide garbage under a carpet. Occam would be unhappy with that :)

File: src/katatraining/collatzcojnjecturelength/Collatz.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        int i = 1;
+        do {
+            if (x % 2 == 0) {
+                x = x / 2;
+            } else {
+                x = 3 * x + 1;
+            }
+            i++;
+        }
+        while (x != 1);
```
Comment (ezseeton) [inline src/katatraining/collatzcojnjecturelength/Collatz.java:30]: what will happen if there will be 0 or negative value as the conjecture method parameter? Will it work correctly?

**Review by ezseeton — COMMENTED** [review]
I have some questions

File: test/katatraining/collatzcojnjecturelength/CollatzTest.java (hunk 1 above)
Comment (udodikdmitro) [reply test/katatraining/collatzcojnjecturelength/CollatzTest.java]: It's done!!

File: src/katatraining/collatzcojnjecturelength/Collatz.java (hunk 2 above)
Comment (udodikdmitro) [reply src/katatraining/collatzcojnjecturelength/Collatz.java:30]: Only non-negative values. But I fixed for zero.

**Review by ezseeton — APPROVED** [review]

**Pull request merged**